server/plugin.go

```diff
@@ -282,16 +282,20 @@ func (p *Plugin) start(isRestart bool) {
 		return
 	}
 
-	p.monitor = NewMonitor(p.GetClientForApp(), p.store, p.API, p.GetMetrics(), p.GetURL()+"/", p.getConfiguration().WebhookSecret, p.getConfiguration().EvaluationAPI)
-	if err = p.monitor.Start(); err != nil {
-		p.API.LogError("Unable to start the monitoring system", "error", err.Error())
+	if p.configuration.EnableSyncJob {
+		p.monitor = NewMonitor(p.GetClientForApp(), p.store, p.API, p.GetMetrics(), p.GetURL()+"/", p.getConfiguration().WebhookSecret, p.getConfiguration().EvaluationAPI)
+		if err = p.monitor.Start(); err != nil {
+			p.API.LogError("Unable to start the monitoring system", "error", err.Error())
+		}
+	} else {
+		p.API.LogInfo("Sync job disabled via config")
 	}
 
 	ctx, stop := context.WithCancel(context.Background())
 	p.stopSubscriptions = stop
 	p.stopContext = ctx
 
-	if !p.getConfiguration().DisableCheckCredentials {
+	if !p.getConfiguration().DisableCheckCredentials && p.configuration.EnableSyncJob {
```

Comment:
Why do we need two configurations to disable the credentials job?

Reply:
There is no UI that I can find to `DisableCheckCredentials`   It is only set by unit test helpers.

Reply:
Indeed: should this setting be the same? Disabling this functionality isn't something someone should do for the production version of this plugin. Developers can manipulate this via config.json.

Reply:
I don't think there is a production version of this plugin - at least not one where someone is using the sync feature.  In a few weeks we'll have the requisite code in a separate plugin and I don't think this one will be deployed anywhere.

Reply:
I'm actively using the one-way sync feature of this plugin on Hub. We've prepackaged the functionality and shipped it in the last ESR. And a customer at CKO mentioned using it (although they balked at the cost, so stopped). I can't say with confidence that no one is using this.

Reply:
> Disabling this functionality isn't something someone should do for the production version of this plugin

This would have been a nicely hidden setting as a env var not meant for production use.  That was my original plan.